sympy/core/assumptions.py

```diff
@@ -210,9 +210,8 @@ class StdFactKB(FactKB):
 
     This is the only kind of FactKB that Basic objects should use.
     """
-    rules = _assume_rules
-
     def __init__(self, facts=None):
+        super(StdFactKB, self).__init__(_assume_rules)
```

Comment:
Is this replacing the line removed above?

Reply:
Yes, it makes sense to call the super parent init so it results in the same result as setting rules above.